website/docs/os/os.findheader.md

```diff
@@ -6,14 +6,21 @@ p = os.findheader("headerfile" [, additionalpaths])
 
 ### Parameters ###
 
-`headerfile` is a file name or a the end of a file path to locate.
+`headerfile` is a file name of a file path to locate.
 
-`additionalpaths` is a string or a table of one or more additional search path.
+`additionalpaths` is a string or a table of one or more additional search path. Can be absolute or relative paths. If relative, they are relative to all the default search paths.
```

Comment:
Suggestion: "The paths may be absolute or relative. If the path is a relative path, it is relative to each of the default search paths."